app/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1,3 @@
+constants = co.lateralview.myapp.BuildConfig
+manifest = --none
+application = co.lateralview.myapp.TestApplication
```

Comment:
You can also change those config properties by annotations, like

`@Config(application = TestApplication.class)`

Reply:
Yes, default behavior is defined in this `properties` file.
If you need to override this config for a particular test class you can use `@Config` there